lib/hawkular/alerts/alerts_api.rb

```diff
@@ -81,12 +81,100 @@ def create_trigger(trigger, conditions = [], dampenings = [], _actions = [])
       http_post 'triggers/trigger', full_trigger
     end
 
+    # Creates the group trigger definition.
+    # @param [Trigger] trigger The group trigger to be created
+    # @return [Trigger] The newly created group trigger
+    def create_group_trigger(trigger)
+      ret = http_post 'triggers/groups', trigger.to_h
+      Trigger.new(ret)
+    end
+
+    # Updates a given group trigger definition
+    # @param [String] trigger_id ID of the group trigger to update
+    # @param [Trigger] trigger the group trigger to be updated
+    # @return [Trigger] The updated group trigger
+    def update_group_trigger(trigger_id, trigger)
```

Comment:
why is there a need to pass both trigger id and trigger objects as parameters to this method and not extract the trigger id from the trigger object?